pkg/crc/machine/start.go

```diff
@@ -151,11 +151,11 @@ func growRootFileSystem(sshRunner *crcssh.Runner, preset crcPreset.Preset, persi
 }
 
 func getrootPartition(sshRunner *crcssh.Runner, preset crcPreset.Preset) (string, error) {
-	diskType := "xfs"
+	cmd := "/usr/sbin/blkid --label root -o device"
 	if preset == crcPreset.Microshift {
-		diskType = "LVM2_member"
+		cmd = "/usr/sbin/blkid -t TYPE=LVM2_member -o device"
 	}
-	part, _, err := sshRunner.RunPrivileged("Get device id", "/usr/sbin/blkid", "-t", fmt.Sprintf("TYPE=%s", diskType), "-o", "device")
+	part, _, err := sshRunner.RunPrivileged("Get device id", cmd)
```

Comment:
Might be better to avoid the duplication by using `query := "--label root` and `query = `-t TYPE=LVM2_member` and then hardcoding `/usr/sbin/blkid` and `-o device` in a single place.

Reply:
updated the commit to use the above suggestion!